src/Drivers/Database.php

```diff
@@ -26,14 +31,14 @@ public function prune(Auditable $model): bool
     {
         if (($threshold = $model->getAuditThreshold()) > 0) {
             $forRemoval = $model->audits()
+                ->select('id')
                 ->latest()
-                ->get()
-                ->slice($threshold)
-                ->pluck('id');
+                ->offset($threshold)->limit((int) '18446744073709551615')
```

Comment:
What is this? :)

Reply:
Ok, larastan says that we are bringing unnecessary data to memory.
Example, if we have `$threshold = 1000` and audits are 1001, we brings to memory 1001 colection and slide just last one
maybe it would be better to open another PR to discuss this change, i can revert this and set an exception for phpstan
I imagine this goes for V14

Reply:
Other better way just delete, without checking empty
```php
return $model->audits()
      ->select('id')
      ->latest()
      ->offset($threshold)->limit((int) '18446744073709551615')
      ->delete() > 0;
```

Reply:
This change was added here [_(12ded01)_](https://github.com/owen-it/laravel-auditing/commit/12ded011b140140fea3f8ae49ccc45493aff9eff ), any reason to do it like this?